packages/example/components/context/ContextFactory.tsx

```
@@ -0,0 +1,79 @@
+    const dispatch = useCallback((action: Action<T>) => {
+      setState(action);
+    }, []);
+
+    // 为每个字段创建独立的 Provider
+    const providers = useMemo(() => {
+      return Object.keys(state).reduce((acc, key) => {
+        const Context = getFieldContext(key as keyof T);
+        return (
+          <Context.Provider value={state[key as keyof T]}>
+            {acc}
+          </Context.Provider>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)